chromedp_test.go

```diff
@@ -642,30 +643,68 @@ func TestDownloadIntoDir(t *testing.T) {
 	ctx, cancel := testAllocate(t, "")
 	defer cancel()
 
+	// Wrap the context into a timeouted context, to ensure test with not run endless if download fails
+	// A second should be enough for this mocked download to finish, even on slower machines
+	ctx, cancel = context.WithTimeout(ctx, 1*time.Second)
+	defer cancel()
+
```

Comment:
Let's remove this part. Let the go test tool handle test timeout.

Reply:
Alright, didn't know, this is something integrated into the tooling!